cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -895,6 +905,8 @@ second of the three MapReduce programs, which is where it would have left off wh
 With workflows, *suspend* and *resume* require a *run-id* as the action takes place on
 either a currently running or suspended workflow.
 
+As a workflow is initially deployed in a *suspended* state, a call to this API is needed to *resume* it.
```

Comment:
This doesn't sound right. When an app is deployed Workflow like any other program is stopped. Now either an user can manually start it or its associated schedule will trigger it.